skimage/measure/segmentation_metrics.py

```diff
@@ -0,0 +1,493 @@
+import numpy as np
+import multiprocessing
+from .simple_metrics import _assert_compatible
+import scipy.sparse as sparse
+from scipy.ndimage.measurements import label
+
+__all__ = [ 'compare_adapted_rand_error',
+            'compare_rand_by_threshold',
+            'compare_raw_edit_distance',
+            'compare_split_vi',
+            'compare_vi',
+          ]
+
+def compare_raw_edit_distance(im_true, im_test, size_threshold=1000):
+    """Compute the edit distance between two segmentations.
```

Comment:
I think for metrics it is highly important to provide a comprehensive description and include the corresponding formula's.

Reply:
Yes I agree. Actually I'm not positive that this particular metric computation is correct, so I suggest we remove it for now.

Reply:
Having said that, for the other metrics, please provide references to papers or other implementations. They are mostly easy to find. =)